packages/windows/vcruntime/ucrt-readme.txt

```diff
@@ -0,0 +1,53 @@
+Subject to MICROSOFT WINDOWS SOFTWARE DEVELOPMENT KIT (SDK) FOR WINDOWS 10 [1], This package
+ contains Universal C Runtime files as redistributable code:
+
+[1] https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk
+
+See sdk_license.rtf in SDK section 2.a DISTRIBUTABLE CODE, "ii. Distribution Requirements" and

```

Comment:
Is this OK without a link to the license file(sdk_license.rtf)?

Reply:
I've searched license text online, but can't find it.